easypost/endshipper.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from easypost.easypost_object import (
+    EasyPostObject,
+    convert_to_easypost_object,
+)
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.resource import AllResource
+
+
+class EndShipper(AllResource):
+    @classmethod
+    def create(cls, api_key: Optional[str] = None, **params) -> "EndShipper":
+        """Create an endshipper."""
```

Comment:
We should match the proper name

```suggestion
        """Create an EndShipper."""
```

Reply:
resolved